tools/Customization/DevHome.Customization/ViewModels/DevDriveInsights/DevDriveCardViewModel.cs

```diff
@@ -31,6 +38,47 @@ public partial class DevDriveCardViewModel : ObservableObject
 
     public string DevDriveFreeSizeText { get; set; }
 
+    public bool IsDevDriveTrusted { get; set; }
+
+    public string DevDriveTrustText { get; set; }
+
+    public string UntrustedDevDriveDescription { get; set; }
+
+    public string UntrustedDevDriveText { get; set; }
+
+    public string MakeTrustedText { get; set; }
+
+    public char DriveLetter { get; set; }
+
+    [RelayCommand]
+    private void MakeDevDriveTrusted()

```

Comment:
Yes, I am not auto-refreshing the page if that is what you are asking.

Reply:
I'm asking, because if an error occurs in `try` then DevDrive might not be trusted.  One example is if the user declines the UAC prompt.

Reply:
If the user declines the UAC prompt or we somehow fail, as I mentioned, the UX does not update right away and would stay untrusted. Next time you refresh, it would fetch the new data and continue to stay untrusted.